mediorum/server/server.go

```diff
@@ -266,8 +267,9 @@ func New(config MediorumConfig) (*MediorumServer, error) {
 		isAudiusdManaged: isAudiusdManaged,
 
 		peerHealths:        map[string]*PeerHealth{},
-		redirectCache:      imcache.New(imcache.WithMaxEntriesOption[string, string](50_000)),
-		uploadOrigCidCache: imcache.New(imcache.WithMaxEntriesOption[string, string](50_000)),
+		redirectCache:      imcache.New(imcache.WithMaxEntriesLimitOption[string, string](50_000, imcache.EvictionPolicyLRU)),
+		uploadOrigCidCache: imcache.New(imcache.WithMaxEntriesLimitOption[string, string](50_000, imcache.EvictionPolicyLRU)),
+		imageCache:         imcache.New(imcache.WithMaxEntriesLimitOption[string, []byte](1_000, imcache.EvictionPolicyLRU)),
```

Comment:
1k is small, can we do 5 or 10K? average size is probably < 100kb (most requests being for the 150x150 images)
wdyt?